util/helpfulIO.py

```diff
@@ -101,22 +118,29 @@ def _setSimMotor(self, amount: float) -> None:
             if SmartDashboard.getBoolean(
```

Comment:
even just in sim all this smartdash can't be good

Reply:
This PR is just for pure cleanup. Functional changes can come in later PRs